api/src/utils/mailer-templates.ts

```diff
@@ -0,0 +1,169 @@
+import { IUser, IPitch, IPitchAggregate } from 'ssw-common';
+import User from '../models/user';
+
+export interface EmailMessage {
```

Comment:
Does nodemailer not have importable types rather than hardcoding these types ourselves?